outputs.tf

```diff
@@ -162,52 +162,76 @@ output "shell" {
-      OBJECT_STORAGE_BUCKET_RUN_LOGS                 = module.storage.run_logs_bucket,
-      OBJECT_STORAGE_BUCKET_STATES                   = module.storage.states_bucket,
-      OBJECT_STORAGE_BUCKET_USER_UPLOADED_WORKSPACES = module.storage.user_uploaded_workspaces_bucket,
-      OBJECT_STORAGE_BUCKET_WORKSPACE                = module.storage.workspace_bucket,
-      OBJECT_STORAGE_BUCKET_METADATA                 = module.storage.metadata_bucket
-      OBJECT_STORAGE_BUCKET_UPLOADS                  = module.storage.uploads_bucket
-      OBJECT_STORAGE_BUCKET_UPLOADS_URL              = "https://storage.googleapis.com"
-
       # Database
-      DATABASE_NAME            = var.enable_database ? module.db[0].database_name : ""
-      DATABASE_USER            = var.enable_database ? module.db[0].database_iam_user : ""
-      DATABASE_CONNECTION_NAME = var.enable_database ? module.db[0].database_connection_name : ""
-      DB_ROOT_PASSWORD         = var.enable_database ? module.db[0].database_root_password : ""
+      DATABASE_NAME    = var.enable_database ? module.db[0].database_name : ""
+      DATABASE_USER    = var.enable_database ? module.db[0].database_iam_user : ""
+      DB_ROOT_PASSWORD = var.enable_database ? module.db[0].database_root_password : ""
 
       #GKE
       GKE_CLUSTER_NAME = module.gke.gke_cluster_name
     },
   })
 }
+
+output "kubernetes_secrets" {
+  sensitive   = true
+  description = "Kubernetes secrets required for the Spacelift services. This output is just included as a convenience for use as part of the EKS getting started guide."
+  value = templatefile("${path.module}/kubernetes-secrets.tftpl", {
+    NAMESPACE                                      = var.k8s_namespace
+    SERVER_DOMAIN                                  = var.website_domain
+    MQTT_BROKER_DOMAIN                             = module.dns.mqtt_endpoint
+    GCP_PROJECT                                    = var.project,
+    OBJECT_STORAGE_BUCKET_DELIVERIES               = module.storage.deliveries_bucket
+    OBJECT_STORAGE_BUCKET_LARGE_QUEUE_MESSAGES     = module.storage.large_queue_messages_bucket
+    OBJECT_STORAGE_BUCKET_MODULES                  = module.storage.modules_bucket
+    OBJECT_STORAGE_BUCKET_POLICY_INPUTS            = module.storage.policy_inputs_bucket
+    OBJECT_STORAGE_BUCKET_RUN_LOGS                 = module.storage.run_logs_bucket
+    OBJECT_STORAGE_BUCKET_STATES                   = module.storage.states_bucket
+    OBJECT_STORAGE_BUCKET_USER_UPLOADED_WORKSPACES = module.storage.user_uploaded_workspaces_bucket
+    OBJECT_STORAGE_BUCKET_WORKSPACE                = module.storage.workspace_bucket
+    OBJECT_STORAGE_BUCKET_METADATA                 = module.storage.metadata_bucket
+    OBJECT_STORAGE_BUCKET_UPLOADS                  = module.storage.uploads_bucket
+    DATABASE_URL                                   = var.enable_database ? "postgres://${module.db[0].database_iam_user}@127.0.0.1/${module.db[0].database_name}" : ""
```

Comment:
Just double checking, is our thought process here that if someone wants to bring their own DB, they won't be using this output and will be ok to setup the secret fully themselves? That's fine if so - just making sure it's a conscious decision.

Reply:
My idea is that if you want to bring your own bottles to the party, you can still use this output, but adjust it before applying to inject your custom values.
That's why in the guide I switched to piping the output to a file instead of kubectl directl.

See here https://github.com/spacelift-io/user-documentation-shv3/pull/61/files#diff-670d491e01ba90ee72ed0ef3254163447585c92ed643f1898dc88080f727c5b1R491-R497

Reply:
Yeah - I saw the comment in the guide - it's just I reviewed this module first and forgot to come back and update the comment. Makes sense.